src/analyses/invariant_set.cpp

```diff
@@ -374,6 +374,7 @@ void invariant_sett::add_type_bounds(const exprt &expr, const typet &type)
   {
     std::size_t op_width=to_unsignedbv_type(type).get_width();
 
+    // TODO - 8 appears to be a magic number here
```

Comment:
It would be good to name these TODOs `// TODO(tautschnig): ...` also create an issue as otherwise this would probably be lost

Reply:
Logged as #1707.